dissect/target/plugins/general/plugins.py

```diff
@@ -87,7 +87,7 @@ def get_description_dict(
     output_descriptions = []
     for key in structure_dict.keys():
         output_descriptions += [
-            textwrap.indent(key + ":", prefix=" " * indentation_step)
+            textwrap.indent(key + ":", prefix=" " * indentation_step) if key != "" else "OS plugins (no namespace)"
```

Comment:
```suggestion
            textwrap.indent(key + ":", prefix=" " * indentation_step) if key != "" else "OS plugins"
```
I don't believe it's necessary to display this.